tutorials/best_practices/scenes_versus_scripts.rst

```diff
@@ -39,8 +39,8 @@ a change in API:
 
     public partial class Game : Node
     {
-        public readonly Script MyNodeScr = (Script)ResourceLoader.Load("MyNode.cs");
-        public readonly PackedScene MySceneScn = (PackedScene)ResourceLoader.Load("MyScene.tscn");
+        public Script MyNodeScr { get; } = GD.Load<Script>("MyNode.cs");
+        public PackedScene MySceneScn { get; } = GD.Load<PackedScene>("MyScene.tscn");
```

Comment:
Considering how these are `const` in GDScript, I would make these properties `static`.

I would also drop the `Scr`/`Scn` suffix, and make the fields below private and renamed them to use `_camelCase`.

```suggestion
        public static Script MyNode { get; } = GD.Load<Script>("MyNode.cs");
        public static PackedScene MyScene { get; } = GD.Load<PackedScene>("MyScene.tscn");
```